### DISCUSSION TITLE
Rendering warning sign for unconnected nodes and handles

### DISCUSSION THREAD

**Discussion by mmexploit** [xyflow/xyflow#2882] [Q&A]
I wanted to render a warning sign whenever a particular node has no connection and is floating around. Is there any way I could know dynamically when a handle or node has no connection?

**Comment by moklick** [comment]
you could check that by accessing the store in your custom node for example:

```js 
const hasConnection = useStore(s => s.edges.some(edge => edge.source === id || edge.target === id));
```
